extension/commercial_data/README.md

```diff
@@ -0,0 +1,398 @@
+
+Therefore, the client must have an ability to search the available products that match their desired criteria and inspect their metadata to decide whether the products should be purchased.
+
+The client can then create an order for the desired products. Because of the financial cost of purchasing the data, a separate endpoint for confirming the execution of that specific order should be implemented.
+
+When the order is completed, the user should be able to access the data using the `collection_id` at the temporal and spatial location of the purchased products.
+
+### Collection discovery
+
+A backend should add general information about a commercial data collection to the `/collections` and `/collections/{collection_id}` endpoints, the same as with freely available collections. Only the metadata about the entire dataset needs to be provided, not about the specific data products that a user has already purchased. 
+
+Commercial data collections are distinguished from freely available collections by including `"order:status": "orderable"` as specified in the [STAC Order specification](https://github.com/stac-extensions/order).
+
+Commercial data collections can include an `order_parameters` field if ordering supports additional parameters that specify how the products should be delivered.
+
+Commercial data collections must also include human-readable pricing information for searching and ordering the products. If searching the products is free it should be set to `null`. If needed, references to additional information about pricing can be added to `links`.
+
+#### Example
+
+```json
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/datacube/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/order/v1.0.0/schema.json"
+  ],
+  "type": "Collection",
+  "id": "PLEIADES",
+  "title": "Airbus Pleiades",
+  "description": "Pleiades is a satellite constellation providing very high-resolution optical imagery and is owned by Airbus. Pléiades is composed of two twin satellites orbiting the Earth 180° apart. The satellites deliver 0.5 m optical imagery and offer a daily revisit capability to any point on the globe.",
+  "license": "proprietary",
+  "providers": [...],
+  "extent": {...},
+  "cube:dimensions": {...},
+  "summaries": {...},
+  "assets": {...},
+  "order:status": "orderable",
+  "order_parameters": [
+      {
+        "name": "sample_type",
+        "description": "sample type of the output raster.",
+        "schema": {
+          "type": "string"
+        },
+        "enum": [
```

Comment:
The enum should go into the schema, right?

Reply:
Yeah, if `order_parameters` should basically be defined as `process_parameters` in `GET /service_types`, then you'd put the enum into the schema.

Reply:
Fixed.